cli/src/commands/schema/edit.ts

```diff
@@ -432,26 +456,69 @@ Beware that this can lead to ${chalk.bold(
         {
           name: 'unique',
           message: 'Whether the column is unique (true/false)',
-          validate: validateOptionalBoolean
+          validate(value: string | undefined, state: ColumnEditState, item: unknown, index: number) {
+            if (!isColumnAdded && parseBoolean(value) !== parseBoolean(state.initial.unique)) {
+              return `Cannot change Unique for existing columns`;
+            }
+            const validateOptionalBooleanResult = validateOptionalBoolean(value);
+            if (validateOptionalBooleanResult !== true) {
+              return validateOptionalBooleanResult;
+            }
+            const validateUniqueResult = validateUnique(value, state);
+            if (validateUniqueResult !== true) {
+              return validateUniqueResult;
+            }
+            return true;
+          }
         },
         {
           name: 'notNull',
           message: 'Whether the column is not nullable (true/false)',
-          validate(value: string, state: ColumnEditState, item: unknown, index: number) {
-            return validateOptionalBoolean(value);
+          validate(value: string | undefined, state: ColumnEditState, item: unknown, index: number) {
+            if (!isColumnAdded && parseBoolean(value) !== parseBoolean(state.initial.notNull)) {
+              return `Cannot change Not null for existing columns`;
```

Comment:
[copy] I noticed in errors you're capitalizing Not null, Default Value, Unique, Description. Probably they should be lowercase inside the error message

Reply:
I'd attempted to match it to what the user sees on the left of each field (I'd messed up and put a capital V on value though). There are three places they're shown
<img width="651" alt="Screenshot 2023-02-17 at 07 48 52" src="https://user-images.githubusercontent.com/6640874/219584075-c92198f6-e678-4b49-b8d2-2353ac36d2d4.png">

I'm thinking we should make them all lower camel case in all three places? WDYT?

<img width="555" alt="Screenshot 2023-02-17 at 07 54 31" src="https://user-images.githubusercontent.com/6640874/219584557-4e19c2fc-8c2e-4645-97fa-9b88e6aa48f7.png">

Reply:
Sounds good to me the, if it's a reference to something else maybe we make it look a bit different with chalk instead?

Reply:
We can always follow up with any chalk stuff - I think it's good enough for now

https://github.com/xataio/client-ts/pull/909/commits/d6a37ef8da243c94af633153715c60ddbeba4659